### DISCUSSION TITLE
Backup not sticking

### DISCUSSION THREAD

**Discussion by Telemachus1977** [tagyoureit/nodejs-poolController#655] [App and O/S Config Questions]
Not sure why, but this morning both my njsPC and REM were reset to default settings.

I had a backup saved, so I restored both.

njsPC is back how it was, but REM won't take any of the settings.

Is there a way to manually copy/paste the settings?

**Comment by rstrouse** [comment]
Yes in the backup there is a file called controllerConfig.json.  Shut down REM and copy that file to the `~/relayEquipmentManager/data/` directory.

**Reply by Telemachus1977** [reply]
Thank you. I feel like the same issue that was not allowing uploads to interface rules is blocking this from working as well.  I have tried changing the attributes of the folders to make them more accessible, but it does not change anything.

**Reply by rstrouse** [reply]
When you run PM2 are you using sudo?  Permissions in *nix spin me out sometimes.

**Reply by tagyoureit** [reply]
Or if you installed REM with sudo you may not have the right permissions, either.  You should look at the owner/permissions with `ls -al` and see who owns the files/directories.

**Reply by Telemachus1977** [reply]
All possible.  Thanks for the ideas
